src/tools/genpolicy/src/registry.rs

```diff
@@ -232,7 +232,10 @@ async fn get_image_layers(
     for layer in &manifest.layers {
         if layer
             .media_type
-            .eq(manifest::IMAGE_DOCKER_LAYER_GZIP_MEDIA_TYPE)
```

Comment:
Is manifest::IMAGE_LAYER_GZIP_MEDIA_TYPE support a new feature of the latest oci_distribution?

- If it's an already existing feature, I think you should use a separate PR for this change.
- If it's new, then using a single commit seems reasonable to me.